build.gradle.kts

```
@@ -22,6 +22,11 @@ allprojects {
         failBuildOnCVSS = 9.0F
         suppressionFile = "suppressions.xml"
         nvd.apiKey = System.getenv("NVD_API_KEY")
+        analyzers {
+            nodeAudit {
+                enabled = false
```

```suggestion
                enabled = false
                yarnEnabled = false
```

Locally I was still getting the yarn error so I think we need this as well. Getting 403'd now so I can't retest